include/deal.II/fe/fe.h

```diff
@@ -1469,7 +1469,7 @@ class FiniteElement : public Subscriptor, public FiniteElementData<dim>
    */
   std::pair<unsigned int, unsigned int>
   face_system_to_component_index(const unsigned int index,
-                                 const unsigned int face_no = 0) const;
+                                 const unsigned int face = 0) const;
```

Comment:
Is this change intentional? It seems to contract the other `face_no` argument names?

Reply:
I have reverted this change.